RELEASES.md

```diff
@@ -6,9 +6,164 @@ services and the knowledge graph of Thoth.
 Release cycles may vary from 2 to 4 weeks, depending on complexity of features or capacity of the team! For the current
 state of planning, have a look at our [Projects](https://github.com/orgs/thoth-station/projects).
 
+## Release 2021.02.01
```

Comment:
We have also new configuration options propagated from .thoth.yaml. As they are not used in the recommendation engine itself, not sure if it is worth to be stated.

Reply:
lets keep them for a post when we can show them in action!